internal/configuration/validator/const.go

```diff
@@ -213,3 +220,7 @@ const testTLSCert = "/tmp/cert.pem"
 const testTLSKey = "/tmp/key.pem"
 
 const errAccessControlInvalidPolicyWithSubjects = "Policy [bypass] for domain %s with subjects %s is invalid. It is not supported to configure both policy bypass and subjects. For more information see: https://www.authelia.com/docs/configuration/access-control.html#combining-subjects-and-the-bypass-policy"
+
+const errOAuthOIDCServerHMACLengthMustBe32Fmt = "OIDC Server HMAC secret must be exactly 32 chars long but is %d long"
```

Comment:
I think we should not bother the user with that kind of constraint, let's hash the secret into a 32 bytes hash. That's gonna be simpler. SHA-256 would do the trick.

Reply:
Yeah that makes sense. Do we want to force the client secret to be a bcrypt hash though? I think for the sake of client secrets it's probably better, but most other providers don't do it. Alternatively we could dynamically detect it and warn if people put plain text secrets in their config.

Reply:
Done.